src/main/java/seedu/address/model/Model.java

```diff
@@ -28,11 +27,10 @@ public interface Model {
 
     /** {@code Predicate} that evaluate to true for all unarchived applications */
     Predicate<InternshipApplication> PREDICATE_SHOW_ONGOING_APPLICATIONS = internshipApplication ->
-            internshipApplication.getStatus() != InternshipStatus.ARCHIVED;
+            !internshipApplication.archived();
```

Comment:
Personally I think we can display only archived applications in the list command, but for the find command maybe we could stick to listing all applications, since there's a possibility someone would like to look for an archived application using the search function